sqlobject/src/main/java/org/jdbi/v3/sqlobject/DefaultObjectBinder.java

```diff
@@ -21,13 +21,19 @@
 import org.jdbi.v3.core.SqlStatement;
 import org.jdbi.v3.core.util.GenericTypes;
 
-class DefaultObjectBinder implements Binder<Bind, Object>
+class DefaultObjectBinder implements BinderFactory<Bind, Object>, Binder<Bind, Object>
 {
     DefaultObjectBinder()
     {
     }
 
     @Override
+    public Binder<Bind, Object> build(Bind annotation)
+    {
+        return this;
```

Comment:
Could do what the other binder factories are doing, and turn the `bind` method into a lambda that we return from the `build` method.